templates/alert.html

```diff
@@ -1,6 +1,8 @@
+<style><% include ../assets/css/cd-alert.css %></style>
+
 <% if (typeof alert !== 'undefined' && alert.type && alert.message) { %>
-  <div class="alert alert--inline alert--<%= alert.type %> [ flow ]" role="alert">
-    <div class="alert__inner">
+  <div class="cd-alert cd-alert--inline cd-alert--<%= alert.type %> [ flow ]" role="alert">
```

Comment:
we can totally remove `cd-alert--inline` — I'm not sure where/when it was being used, but it's officially cruft now.

Reply:
I popped a commit onto the branch